### PR TITLE
updated packages

### PR DESCRIPTION
**Pull request by eeerin**
https://structure.atlassian.net/browse/SP-4801

### CODE REVIEW

File: lib/model.js [hunk 1]
```diff
@@ -4,12 +4,12 @@ const etag = require('etag');
 const debug = require('debug')('cosa:model');
 const objectPath = require('object-path');
 const defaults = require('defaults');
-const clone = require('clone');
+// const clone = require('clone');
```
Comment (spacetc62) [inline lib/model.js]: remove commented code?

File: lib/model.js [hunk 2]
```diff
@@ -65,7 +65,7 @@ const _buildSchema = (definition) => {
       }
     }
   }
-  return schema;
+  return joi.object(schema);
```
Comment (eeerin) [inline lib/model.js]: https://github.com/hapijs/joi/issues/2037

File: lib/model.js [hunk 3]
```diff
@@ -268,7 +268,7 @@ const _create = (data, definition) => {
   definition.methods._validate = async function(obj, options = {}) {
     const joiOptions = defaults(onlyJoiOptions(options), defaultJoiOptions);
     try {
-      return await joi.validate(obj, definition._schema, joiOptions);
+      return await definition._schema.validateAsync(obj, joiOptions);
```
Comment (eeerin) [inline lib/model.js]: https://github.com/hapijs/joi/issues/2037

File: lib/utils.js [hunk 4]
```diff
@@ -66,7 +66,7 @@ const buildPropertySchema = (name, propertyDef) => { // eslint-disable-line comp
       if (propertyDef.trim) { schema = schema.trim(); }
       break;
     case 'objectid':
-      schema = joi.object().type(ObjectID);
+      schema = joi.object().instance(ObjectID);
```
Comment (eeerin) [inline lib/utils.js]: https://github.com/hapijs/joi/issues/2037 type -> instance

File: lib/utils.js [hunk 5]
```diff
@@ -87,7 +87,7 @@ const buildPropertySchema = (name, propertyDef) => { // eslint-disable-line comp
   if ('undefined' !== typeof propertyDef.strict) { schema = schema.strict(propertyDef.strict); }
   if ('undefined' !== typeof propertyDef.label) { schema = schema.label(propertyDef.label); }
   if (propertyDef.raw) { schema = schema.raw(); }
-  if ('undefined' !== typeof propertyDef.default) { schema = schema.default(propertyDef.default, 'default'); }
+  if ('undefined' !== typeof propertyDef.default) { schema = schema.default(propertyDef.default); }
```
Comment (eeerin) [inline lib/utils.js]: you dont need that second arg any more

**Pull request merged**